web/static/css/objects/_navigation.css

```diff
@@ -1,60 +1,172 @@
+$header_background_color: #fff;
+
 .so-navigation {
-  list-style: none;
-  margin-top: 0.2em;
-  vertical-align: top;
-  font-size: 0.9em;
-  margin-bottom: 0.3em;
-  padding-top: 0.5em;
+  position: static;
+  width: 100%;
+  max-width: 48em;
+  z-index: 3;
+  background-color: $header_background_color;
+  min-height: 64px;
 }
 
-.so-navigation-item {
-  vertical-align: top;
+.so-navigation__left {
+  display: block;
+  float: left;
+  margin-left: 1em;
+  margin-bottom: 0;
 }
 
-.so-navigation--logo {
+.so-navigation__logo {
   margin-top: -1.5em;
   width: auto;
   height: 6em;
   position: absolute;
+  border: 0;
 }
 
-.so-navigation-item--image * img {
-  width: 2.5em;
-}
-
-.so-navigation--about--no-logo {
+.so-navigation__about--no-logo {
   margin-top: 0.5em;
   color: $deactive;
   font-size: 0.9em;
   display: inline-block;
 }
 
-.so-navigation--about {
+.so-navigation__about {
   margin-left: 6em;
   margin-top: 0.5em;
-  margin-bottom: 3em;
   color: $deactive;
   font-size: 0.9em;
   display: inline-block;
 }
 
+.so-navigation__menu {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  overflow: hidden;
+  background-color: $header_background_color;
+  font-size: 0.9em;
+  clear: both;
+  max-height: 0;
+}
+
+.so-navigation__menu > li > a,
+.so-navigation__menu > li > form {
+  display: block;
+  padding: 0.75em 1em;
```

Comment:
This should be `0.2em 0.3em`

And `@media (min-width: 48em) _searchbar.css:57 .so-searchbar` should be changed to `margin: -1.1em -1em 1em;`

To make the upper logo line up with the menu below it, the `_navigation.css:13 .so-navigation__left` should be changed to have `margin-left: 0em;` if the screen is small enough to show the hamburger